### DISCUSSION TITLE
[Suggestion] Hotkey for focusing on Root/Parent node of scene

### DISCUSSION THREAD

**Discussion by asperkraken** [godotengine/godot-proposals#6520] [Editor]
[EDITED FOR CLARITY] When making edits, it would be very helpful to have a hotkey to return focus to the root node or main parent node of a scene. This could be helpful in preventing many errors in Godot (like when dropping objects into the wrong tree in a scene.)

**Comment by Calinou** [comment]
Could you create a mockup or record a video to describe how this would look in practice? I don't understand what you mean by "top"/"parent" node of a scene here.

**Reply by asperkraken** [reply]
Sorry - I meant to say "root." Ah insomnia.

**Reply by asperkraken** [reply]
To translate: when in the editor, there would be a hotkey to select/focus on the root node of a scene. 

Example: currently in Godot, when you add a node to the scene, that node becomes the selected/focused node in the editor until another is selected. My proposal is to have a hotkey command to instantly change selection/focus to the root node. Should be helpful in editor in preventing objects/scenes being dropped into the wrong part of a tree.

**Comment by AThousandShips** [comment]
By top/home/main do you mean root?

**Reply by AThousandShips** [reply]
If so this would probably be a better addition to the `Tree` class and have it select the root of the tree (if root is visible) on some key like `ui_home`

**Reply by asperkraken** [reply]
Sorry i meant "root," yes!
